src/IECoreImage/DisplayDriverServer.cpp

```diff
@@ -423,9 +437,39 @@ void DisplayDriverServer::Session::handleReadOpenParameters( const boost::system
 
 		const StringData *displayType = parameters->member<StringData>( "remoteDisplayType", true /* throw if missing */ );
 
+		const BoolData *mergeDriverData = parameters->member<BoolData>( "mergeDriver", false);
+        m_mergeSession = mergeDriverData && mergeDriverData->readable();
+
 		// create a displayDriver using the factory function.
-		m_displayDriver = DisplayDriver::create( displayType->readable(), displayWindow->readable(), dataWindow->readable(), channelNames->readable(), parameters );
+		if (!m_mergeSession)
+		{
+			m_displayDriver = DisplayDriver::create( displayType->readable(), displayWindow->readable(), dataWindow->readable(), channelNames->readable(), parameters );
+		}
+		else
+		{
+			m_mergeId = parameters->member<IntData>( "sessionId", true /* throw if missing */ )->readable();
```

Comment:
The parameter list for displays is full of all sorts of stuff we've accumulated over the years, and it's not always clear what is used by who, and when. Perhaps we could call this `displayDriverServer:mergeId` to make it clearer who consumes it, and what it means? And perhaps we could do similar with `sessionClients`?